examples/1d_stencil/1d_stencil_8.cpp

```diff
@@ -353,6 +357,18 @@ struct stepper_server : hpx::components::component_base<stepper_server>
     HPX_DEFINE_COMPONENT_ACTION(stepper_server, from_right, from_right_action);
     HPX_DEFINE_COMPONENT_ACTION(stepper_server, from_left, from_left_action);
 
+    // release dependencies
+    void release_dependencies()
+    {
+        left_ = hpx::shared_future<hpx::id_type>();
+        right_ = hpx::shared_future<hpx::id_type>();
```

Comment:
`hpx::find_from_basename()` returns `future<id_type>`. Do you suggest to call `get()` on the returned futures?